builtins/amp-ad.md

```diff
@@ -67,19 +67,11 @@ Optional attribute to pass configuration to the ad as an arbitrarily complex JSO
 An `amp-ad` must have static layout defined as is the case with any other AMP element. However,
 it's possible to resize an `amp-ad` in runtime. To do so:
 
-1. The `amp-ad` must be defined with `resizable` attribute;
-2. The `amp-ad` must have `overflow` child element;
-3. The Ad's Iframe document has to send a `embed-size` request as a window message.
+1. The Ad's Iframe document has to send a `embed-size` request as a window message.
```

Comment:
I think we can just delete this here. No need to document how this works to users.